todo_app/data/logs/app.log

```diff
@@ -157,3 +157,141 @@
 ### 2022-03-02 02:49:23,086 ### appLog ### INFO ### Card due date: 07/12/2022 ###
```

Comment:
We chatted through removing this already, but just to add I think that's a good shout but also nice work on getting a log file set up!